.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.9.5
```

Comment:
Is bumping version for such change is common practice? If yes the everything else LGTM.

Reply:
I've used patch level (this is a patch), but there were a few PRs with no bump.